### PR TITLE
Ticket 5 core get api articles id comments

### PR DESCRIPTION
**Pull request by robharmer3**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -129,4 +129,54 @@ describe("GET /api/articles", () => {
...
     })
   })
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with a array of comments of the given article ID", () => {
+    return request(app)
+    .get("/api/articles/3/comments")
+    .expect(200)
+    .then(({ body }) => {
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:139]: Could check all comments with a loop, also to make sure all the comments are contained in the response.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -129,4 +129,54 @@ describe("GET /api/articles", () => {
...
+      expect(body.comments[0]).toMatchObject({
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:140]: Good use of toMatchObject

File: __tests__/app.test.js [hunk 3]
```diff
@@ -129,4 +129,54 @@ describe("GET /api/articles", () => {
...
+    })
+  })
+  test("400: Responds with an bad request error when given a invalid ID", () => {
+    return request(app)
+    .get("/api/articles/darthvader/comments")
+    .expect(400)
+    .then(({ body }) => {
+      expect(body.msg).toBe("Bad Request, invalid input")
+    })
+  })
```
Comment (nc-bela-bertalan) [inline __tests__/app.test.js:181]: Please add one more test where there are no comments for an article, expecting an empty array and 200. 

You will need to check the reason for the query returning empty rows:
article exists but has no comments.
article does not exist.

**Review by nc-bela-bertalan — COMMENTED** [review]
Hi Rob, nice work on this one, just need another test. Please see my comments.
